Update aggregate documentation for 6.x (#239)
neo4j/docs-graphql: modules/ROOT/pages/queries-aggregations/aggregations.adoc

----
type Query {
    posts(where: PostWhere, sort: [PostSort!]!, limit: Int, offset: Int,): [Post!]!
    postsConnection(after: String, first: Int, sort: [PostSort], where: PostWhere): PostsConnection!

    users(where: UserWhere, sort: [UserSort!]!, limit: Int, offset: Int,): [User!]!
    usersConnection(after: String, first: Int, sort: [UserSort], where: UserWhere): UsersConnection!
}
----

== Querying aggregations

> Aggregations on a type are provided in the field `aggregat` inside Connection:

Reviewer: ```suggestion
Aggregations on a type are provided in the field `aggregate` inside Connection:
```

[source, graphql, indent=0]
----
query {
    usersConnection(where: { name_STARTS_WITH: "J" }) {
        aggregate {
            count {
                nodes
            }
        }
    }
}
----

> == Aggregate related nodes

Reviewer: ```suggestion
== Aggregating related nodes
```

> Related nodes can also be aggregated within a query by accessing the aggregation fields in the connection.

Reviewer: ```suggestion
You can aggregate related nodes in a query by accessing the aggregation fields in the connection.
```

In these fields, you can **count**, aggregate the **nodes** or **edges** fields.

> The same selections and types as before are available in relationship aggregations.

Reviewer: ```suggestion
The same selections and types as in xref:#_aggregation_fields[] are available in relationship aggregations.
```

what does "as before" refer to? asking because a link or even repeating the info might make it easier for readers

Author: TBH, this is not new documentation, just updating examples in this PR, but I presume it refers to the section "
== Aggregation fields"

Reviewer: (tricked myself with github views)
updated suggestion
